Package.swift

```diff
@@ -1,10 +1,9 @@
 // swift-tools-version:5.0
 import PackageDescription
 
-// Note: SPM support is broken until we can add dependency on UIKit.
-
 let package = Package(
     name: "Reusable",
+    platforms: [.iOS(.v8), .tvOS(.v9)],
```

Comment:
@MortyMerr what do you think of this suggestion that was made parallel to your PR #89 ?

Since we can now specify platforms on `Package.swift`, it makes more sense to me to list those explicit platforms (and support tvOS) rather than using the `canImport` that you used?

Reply:
You can specify platforms, however it won't help to successfully run `swift build`. `canImport` still needed for that

Reply:
I'm not sure how `swift build` works, I tested this with Xcode. Without my change I got the "no UIKit"-error and with the change it added the framework successfully.

Seems that `swift build` might not utilize the platform which is pretty disappointing and weird since it works for Xcode. https://forums.swift.org/t/swift-build-fails-ignores-platform-description-macos-xcode-beta-6/28133